spec/rubocop/cop/style/method_call_with_args_parentheses_spec.rb

```diff
@@ -935,6 +935,44 @@ def seatle_style arg: default(42)
       RUBY
     end
 
+    context 'endless range literals' do
+      it 'accepts parens when no end node and last argument' do
+        expect_no_offenses(<<~RUBY)
+          foo(2..)
+          foo(1, 2...)
+        RUBY
+      end
+
+      it 'registers an offense no end node and not last argument' do
+        expect_offense(<<~RUBY)
+          foo(2.., 1)
+             ^^^^^^^^ Omit parentheses for method calls with arguments.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          foo 2.., 1
+        RUBY
+      end
+
+      it 'accepts parens when no begin node and first argument' do
+        expect_no_offenses(<<~RUBY)
+          foo(..2, 1)
+          foo(...2)
```

Comment:
Unlike parenthesis-less endless range with subsequent lines, beginless range retains its meaning even without parentheses.

Reply:
Ah, that was my mistake. When `foo(..end_of_range)` becomes `foo ..end_of_range`, the meaning changes.

Reply:
Yup, that's right :+1: . I only noticed because of `Layout/SpaceInsideRangeLiteral` for this specific case.